docs/UserGuide.md

```diff
@@ -97,114 +97,67 @@ Section 2 will explain the features available in easyLog.
+description of those commands.
 
-Format: `items`
+Steps to check all items-related commands:
 
-Expected Output:
-
-```
-items (user input)
-____________________________________________________________
-Items-Related Options:
-  1. items add <item_name>          Add an item
-  2. items list                     List all items
-  3.1 items delete <item_index>     Delete an item
-  3.2 items delete <item_name>      Delete an item
-  4. items clear                    Empty all items
-  5. items find <item_name>         Find an item
-  6. items update                   Update the detail of an item
-  7. items stats                    Get the statistic of items
-____________________________________________________________
-```
+1. Input `items` command.
+2. Press Enter.
+3. All available commands related to items will then be shown.
 
 #### 2.2.2. Add an item: `items add`
 
-Add an item to the system.
-
-Format: `items add <item_name>` followed by `<item_price>` and `<item_stock>`
+When you sometimes overwhelmed by handling a large variety of items, `items add` feature allows you to record the
+details of those items one by one into the easyLog app.
 
-Example: `items add PS5` followed by `100` and `10`
-  (Please note that if an item to be added already exists in the system, additional stock will be requested and the item
-  information will be updated accordingly.)
+Steps to add an item in easyLog:
 
-Expected Output:
-
-```
-items add PS5 (user input)
-____________________________________________________________
-Please input item price and stock.
-Input format: <item_price> <item_quantity>.
-____________________________________________________________
-100 10
-____________________________________________________________
-Got it! The item [PS5] is added.
-____________________________________________________________
-```
+1. Input `items add <item_name>` command.
+2. You will be prompted to key in the item price and the item stock.
+3. Input in this format `<item_price> <space> <item_stock>` for the price and stock to be added to the items details.
+4. Press enter.
+5. You can see the confirmation message when the item is recorded successfully.
 
-💡 item names cannot be empty or consist of only white spaces. <br>
-💡 item price must be a number ranging from 0 to 1,000,000,000 (inclusive). <br>
-💡 item stock has to be a integer ranging from 0 to 1,000,000,000 (inclusive). <br>
+Note:
+> 📄 item names cannot be empty or consist of only white spaces. <br>
+> 📄 item price must be a number ranging from 0 to 1,000,000,000 (inclusive). <br>
+> 📄 item stock has to be an integer ranging from 0 to 1,000,000,000 (inclusive). <br>
+> 📄 When in doubt, do follow the prompts shown on the command terminal. <br>
 
 #### 2.2.3. Show the item list: `items list`
 
-You can use this command to view all the recorded items in the system.
+After your business has been running successfully for a period of time, whenever you would like to check all items in
```

Comment:
Maybe just say whenever you would like to check the items and their stock in your inventory is enough already?